Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -0,0 +1,176 @@
+using Content.Server.Popups;
+using Content.Server.Cargo.Systems;
+using Content.Server.Cargo.Components;
+using Content.Server.Radio.EntitySystems;
+using Content.Server.Shipyard.Components;
+using Content.Server.Shuttles.Components;
+using Content.Server.Shuttles.Systems;
+using Content.Server.Station.Systems;
+using Content.Shared.GameTicking;
+using Content.Shared.Shipyard.Events;
+using Content.Shared.Shipyard.BUI;
+using Content.Shared.Shipyard.Prototypes;
+using Content.Shared.Shipyard.Components;
+using Content.Shared.Access.Systems;
+using Content.Shared.Access.Components;
+using Content.Shared.Shipyard;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+using Robust.Shared.Players;
+using Robust.Shared.Prototypes;
+using Content.Shared.Radio;
+
+namespace Content.Server.Shipyard.Systems
+{
+    public sealed class ShipyardConsoleSystem : SharedShipyardSystem
+    {
+        [Dependency] private readonly AccessReaderSystem _access = default!;
+        [Dependency] private readonly PopupSystem _popup = default!;
+        [Dependency] private readonly UserInterfaceSystem _ui = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] private readonly ShipyardSystem _shipyard = default!;
```

Comment:
The class is named like a partial but is not being treated as such. This means you have 2 systems inheriting SharedShipyardSystem. Just make this a partial of ShipyardSystem.